api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -328,6 +328,8 @@
      */
     int getWindowsFromCheckpoint();
 
+    String getOperatorName();
```

Comment:
+1 for getName. Calls are inside the classes starting with Operator*, methods do not need to contain word *operator*.